operatorcourier/cli.py

```diff
@@ -16,7 +17,10 @@ class _CliParser():
     """
 
     def __init__(self):
-        pass
+        with open('setup.py', 'r') as f:
```

Comment:
shouldn't we only do this in the case that the -v command is specified? rather than during the init